gooddata-sdk/gooddata_sdk/catalog/permission/service.py

```diff
@@ -106,3 +108,56 @@ def manage_dashboard_permissions(
             [permission.to_api() for permission in permissions_for_assignee],
             _check_return_type=False,
         )
+
+    def get_declarative_organization_permissions(self) -> List[CatalogDeclarativeOrganizationPermission]:
+        """Get a list of all declarative organization permissions.
+
+        Args:
+            None
+
+        Returns:
+            [CatalogDeclarativeOrganizationPermission]:
+                List of all declarative organization permissions.
+        """
+
+        catalog_list = []
+        for permission in self._layout_api.get_organization_permissions():
```

Comment:
Please store the result from `self._layout_api.get_organization_permissions()` first and then use it.

e.g.:
`for permission in organization_permissions:`